tests/figures/chart/chart_plugin.test.ts

```diff
@@ -1492,6 +1492,181 @@ describe("title", function () {
       expect(scales.x!["title"].font.style).toEqual("italic");
     }
   );
+
+  test("line chart runtime reflects axis bounds, grids and scale type", () => {
+    setGrid(model, {
+      A1: "Month",
+      A2: "Jan",
+      A3: "Feb",
+      A4: "Mar",
+      B1: "Series A",
+      B2: "5",
+      B3: "15",
+      B4: "25",
+    });
+
+    createChart(
+      model,
+      {
+        type: "line",
+        labelRange: "A2:A4",
+        dataSets: [{ dataRange: "B2:B4" }],
+      },
+      "line-axis"
```

Comment:
nitpick: use an id-like string, w/o knowing what the arguments of `createChart` are, at first glance we're wondering what "line-axis" means. Same in your other tests